stdlib/public/Bionic/Glibc.swift

```diff
@@ -0,0 +1,215 @@
+//===----------------------------------------------------------------------===//
```

Comment:
Please don't duplicate these files.  Can we find some way of reusing the Glibc overlay here, adding some `#if`s into its source?